akka-stream-tests/src/test/java/akka/stream/javadsl/SourceTest.java

```diff
@@ -432,8 +432,8 @@ public void mustBeAbleToUseExpand() throws Exception {
   @Test
   public void mustProduceTicks() throws Exception {
     final TestKit probe = new TestKit(system);
-    Source<String, Cancellable> tickSource = Source.tick(FiniteDuration.create(1, TimeUnit.SECONDS),
-        FiniteDuration.create(500, TimeUnit.MILLISECONDS), "tick");
+    Source<String, Cancellable> tickSource = Source.tick(java.time.Duration.ofSeconds(1),
+            java.time.Duration.ofMillis(500), "tick");
```

Comment:
Wouldn't it be nicer to import `java.time.Duration` and just use `Duration.ofSeconds(1)` here?